backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -482,6 +482,7 @@ public RepositoryEntity setRepository(
         LOGGER.debug("Project role bindings created successfully");
         Transaction transaction = session.beginTransaction();
         repositoryEntity.setCreated(true);
+        repositoryEntity.setVisibility_migration(true);
```

Comment:
This is a completely unrelated fix. Please do not mix unrelated changes in the same PR. We've had a ton of issues with that in the past and it's super confusing for the reviewer.